keyboards/tr60w/keymaps/default/keymap.c

```diff
@@ -1,4 +1,4 @@
-#include "QMK_KEYBOARD_H"
```

Comment:
The CI error was because you had `QMK_KEYBOARD_H` in quotes. It's a macro that gets generated at compile time, and as such, should be unquoted.

This is wrong:

```c
#include "QMK_KEYBOARD_H"
```

This is correct:

```c
#include QMK_KEYBOARD_H
```